source/api.rst

```diff
@@ -127,10 +127,10 @@ The messages returned contain information about the blocks.
 
 * `BlockInfoDTO <https://github.com/nemtech/symbol-openapi/blob/main/spec/core/block/schemas/BlockInfoDTO.yml>`_
 
-**finalized block**
+**finalizedBlock**
 
-The finalized block channel notifies for every subscribed client every time there is a new finalized block.
-The messages returned contain information about the finalized block.
+The ``finalizedBlock`` channel notifies subscribed clients every time a block is finalized.
```

Comment:
It might be a bit confusing, as the server will signal finalization notifications after a number (100 on main/test net, 20 on local dev net) blocks generated. 

perhaps something like: `channel notifies subscribed clients every time the server pushes block finalization notification.`
@fboucquez ?

Reply:
I think both are fine. Steve's is a bit better.

Reply:
i'd say the channel notifies the clients every time finalization takes place, because it does not necessarily happen for each block, rather when a finalization round occurs, every block up to that height/round is finalized

Reply:
I see the problem. The block IS finalized but notifications about it are not sent until some blocks later.
I'm afraid that saying that "the channel notifies clients when the server pushes a notification" is not very informative.
Also, if we say "when finalization takes place", we'll have to explain here what is finalization and when does it take place.

How about we leave that sentence as it is in the PR and we add a comment like: "Bear in mind that the block might already be confirmed (is this the right word?) but it might take a while (X seconds?) until it is finalized during the next finalization round (and we link to the Finalization page that I have to write ASAP)."

Reply:
I'd go with height/round, prefer `round`, something like ` notify clients when the finalization round completed`

Reply:
> I see the problem. The block IS finalized but notifications about it are not sent until some blocks later.
> I'm afraid that saying that "the channel notifies clients when the server pushes a notification" is not very informative.
> Also, if we say "when finalization takes place", we'll have to explain here what is finalization and when does it take place.
> 
> How about we leave that sentence as it is in the PR and we add a comment like: "Bear in mind that the block might already be confirmed (is this the right word?) but it might take a while (X seconds?) until it is finalized during the next finalization round (and we link to the Finalization page that I have to write ASAP)."

not strictly - let me give an example: block 10 finalizes, this means every block up to 10 is finalized, but the blocks before 10 were not necessarily explicitly finalized, so it's not that finalization notifications are delayed, it is that there are finalization rounds where finalization takes place

Reply:
@segfaultxavi Not quite sure I am 100% understand it. My understanding: 
- Finalization round completed, say height 20, notification sent, 
- Latest finalized block height is 20, mean those up to block/20 have been finalized. 
- But, ws notification might only be received at block/24, is that what you mean?

Reply:
OK, after much chatting I think I've got this. Let me make another proposal.